### ISSUE TITLE
💅 typo in rule description

### ISSUE THREAD

**Issue by notramo** [biomejs/biome#1985]
### Environment information

```bash
i Using === may be unsafe if you are relying on type coercion

  i Unsafe fix: Use ===
```

The loose comparison (`==`) operator is used in the code where the error is indicated. It should be "Using == ..." (2 equal signs in the description instead of three)

### Rule name

lint/suspicious/noDoubleEquals

**Comment by ematipico** [comment]
Wanna send a PR?

**Comment by notramo** [comment]
I don't know Rust, and the toolchain is not set up. This seems like a small change, but some tests may fail, and it's not fun to debug them using CI logs.

**Comment by Kenzo-Wada** [comment]
So that I've also faced to this issue, ive created a simple PR !
fell happy if you'd check it :)
